app/controllers/merchant_invoices_controller.rb

```diff
@@ -7,6 +7,6 @@ def index
   def show
     @merchant = Merchant.find(params[:merchant_id])
     @invoice = Invoice.find(params[:invoice_id])
-    @invoiced_item = InvoiceItem.where(invoice_id: params[:invoice_id])
+    #require "pry"; binding.pry
```

Comment:
this can probably be done in a later pull request, but the pry might need to go eventually

Reply:
I'll kill it